public/js/views/management/pay-methods.jsx

```diff
@@ -42,7 +53,7 @@ export default class PayMethods extends Component {
           {this.renderChild()}
           <a className="button quiet add-pay-method" href="#"
             onClick={this.handleAddPayMethod}>{gettext('Add a new card')}</a>
-          <a className="button quiet delete" href="#"
+          <a className={deleteClasses} href="#"
```

Comment:
Sadly disabled attrs don't apply to links: http://www.w3.org/html/wg/drafts/html/master/semantics.html#concept-element-disabled